packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -161,11 +161,14 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
   /** Boolean value to indicate if the role application should be applied on details list. Set to false by default */
   shouldApplyApplicationRole?: boolean;
 
-  /** 
+  /**
    * The minimum mouse move distance to interpret the action as drag event.
    * @defaultValue 5
    */
   minimumPixelsForDrag?: number;
+
+  /** Boolean value to indicate if the component should render in compact mode. Set to false by default */
```

Comment:
Can we explain what compact mode is? And is a boolean good enough, should this be enum for future changes?